### PR TITLE
Add number of meetups to footer for #163

### PR DESCRIPTION
**Pull request by sodevious**

### CODE REVIEW

File: core/context_processors.py [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+from jobs.models import Meetup
 
 
 def statistics(request):
-
     future_events = Event.objects.future()
     past_events = Event.objects.past()
     countries = Event.objects.values('country').distinct()
     attendees = Postmortem.objects.all().aggregate(attendees=Sum('attendees_count'), applicants=Sum('applicants_count'))
+    meetup_count = Meetup.visible_objects.all()
```
Comment (phalt) [inline core/context_processors.py]: You could make this `Meetup.visible_objects.count()` and save an SQL query :)

**Comment by sodevious** [comment]
@phalt Updated

**Comment by phalt** [comment]
Nice! :+1: Looks good to me.

**Comment by asendecka** [comment]
Thank you :sparkles: :heart:

**Pull request merged**